config/operator/base/ui-extensions/serverless/details

```diff
@@ -12,6 +12,9 @@ body:
       - name: Docker Registry
         visibility: $root.spec.dockerRegistry.enableInternal = true
         source: spec.dockerRegistry.enableInternal?"INTERNAL":""
+      - name: Docker Registry Size
```

Comment:
I'm little confused by difference between names `Docker Registry Size` (on details) and `Persistent Volume Size` (on form). Is it intentional?

Reply:
changed to `Persistance Volume Size`